notification-portlet-webapp/src/main/webapp/scripts/jquery.notifications.js

```diff
@@ -66,8 +66,7 @@ var notificationsPortletView = notificationsPortletView || function ($, rootSele
 
     var getDateFormat = function(date) {
       var currentDate = new Date(date);
-      var locale = navigator.language || 'en-US';
-      console.log(locale);
+      var locale = opts.locale || navigator.language || 'en-US';
```

Comment:
is opt.locale user specific or portal wide?
If it is portal wide would:
```suggestion
      var locale = navigator.language || opts.locale || 'en-US';
```

make sense, to prefer the user setting?

Reply:
Edinburgh wanted the ability to force the locale portal wide. I left in the option to let the user's browser drive the value if no portlet preference is set. I expect most organizations will not set `locale` and leave the formatting to the user's preference.